corehq/motech/openmrs/serializers.py

```diff
@@ -74,6 +74,57 @@ def omrs_boolean_to_text(value):
     return 'true' if value else 'false'
 
 
+def unix_timestamp_to_datetime(value, tz=None):
+    """
+    Converts a Unix timestamp to a datetime.datetime instance.
+
+    Given None returns None. Raises ValueError for non-numeric values.
+
+    >>> tz = datetime.timezone(datetime.timedelta(hours=+2), 'CAT')
+    >>> dt = unix_timestamp_to_datetime(1551564000, tz)
+    >>> dt.strftime('%Y-%m-%d')
+    '2019-03-03'
+
+    """
+    if value is None:
+        return None
+    try:
+        timestamp = int(value)
+    except ValueError:
+        raise ValueError(f"{value!r} is not a Unix timestamp")
+    return datetime.datetime.fromtimestamp(timestamp, tz)
+
+
+def posix_milliseconds_to_isoformat(value, tz=None):
+    """
+    Converts an OpenMRS timestamp to ISO format. Accepts a timezone,
+    which defaults to UTC. If timezone is given, ISO format includes the
+    offset, otherwise it ends in "Z".
+
+    An OpenMRS timestamp is a Unix timestamp * 1000 (i.e. with
+    milliseconds). Given None returns None. Raises ValueError for
+    non-numeric values.
+
+    >>> posix_milliseconds_to_isoformat(1551564000000)
+    '2019-03-02T22:00:00Z'
+    >>> tz = datetime.timezone(datetime.timedelta(hours=+2), 'CAT')
+    >>> posix_milliseconds_to_isoformat(1551564000000, tz)
+    '2019-03-03T00:00:00+02:00'
+
+    """
+    if value is None:
+        return None
+    try:
+        timestamp = int(value)
+    except ValueError:
+        raise ValueError(f"{value!r} is not an OpenMRS timestamp")
+    dt = unix_timestamp_to_datetime(timestamp / 1000, tz)
```

Comment:
Are you expecting `unix_timestamp_to_datetime` to be used elswehere?  Otherwise the validation that function provides is redundant - could just drop the function and use the last line here.